cli/src/main/kotlin/commands/ClearlyDefinedUploadCommand.kt

```diff
@@ -115,37 +118,54 @@ class ClearlyDefinedUploadCommand : CliktCommand(
             null
         }
 
+    private fun requestHarvest(request: Collection<ClearlyDefinedService.HarvestRequest>): Boolean =
+        executeApiCall(service.harvest(request)).getOrElse {
+            log.error { it.collectMessagesAsString() }
+        } == HARVEST_CREATED
+
     override fun run() {
         val absoluteInputFile = inputFile.normalize()
         val curations = absoluteInputFile.readValue<List<PackageCuration>>()
-
-        var error = false
-
         val curationsToCoordinates = curations.associateWith { it.id.toClearlyDefinedCoordinates().toString() }
         val definitions = getDefinitions(curationsToCoordinates.values)
 
-        val uploadableCurations = curations.filter { curation ->
-            val harvestStatus = definitions[curationsToCoordinates[curation]]?.getHarvestStatus()
+        val curationsByHarvestStatus = curations.groupBy { curation ->
+            definitions[curationsToCoordinates[curation]]?.getHarvestStatus() ?: log.warn {
+                "Package '${curation.id.toCoordinates()}' has an unknown harvest status and will be ignored."
```

Comment:
I think the user might wonder what "ignored" means here. Will curations not be uploaded? Will no harvest request be sent? If the latter, why not also request it if the status is unknown?

Reply:
I've only changed the wording of the message for now. If @jeffmendoza is right, then requesting the definitions should already start harvesting the belonging components. But it's unclear to be what it means if a definition's coordinates are not in the response. It could mean that the coordinates were already invalid in the request, in which case no harvest would be queued for that component.